docs/run/advanced-runtime-options.mdx

```diff
@@ -1,25 +1,68 @@
 ---
-title: Algorithm tuning options
-description: Common settings to use when tuning primitive options
+title: Advanced runtime options
+description: Specify options when building with primitives 
 
 ---
 
-<span id="tuning-opt"></span>
-# Algorithm tuning options
+# Advanced runtime options
 
+When calling the primitives, you can pass in options by using the `Options` class or when using the `run` method. In the `Options` class, commonly used options, such as `resilience_level`, are at the first level. Other options are grouped into different categories:
 
-One of the advantages of the primitives is that they abstract away the circuit execution setup so that algorithm developers
-can focus on the pure algorithmic components. However, sometimes, to get the most out of an algorithm, you might want
-to tune certain primitive options. This section describes some of the common settings you might need.
+![options](/images/build/options.png)
 
 <Admonition type="info" title="Attention">
-
-    This section focuses on Qiskit Runtime primitive `Options` (imported from ``qiskit_ibm_runtime``). While
-    most of the `primitives` interface is common across implementations, most `Options` are not. Consult the
+    This section focuses on Qiskit Runtime primitive [Options](../api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.Options) (imported from ``qiskit_ibm_runtime``). While most of the `primitives` interface is common across implementations, most `Options` are not. Consult the
     corresponding API references for information about the ``qiskit.primitives`` and ``qiskit_aer.primitives`` options.
 </Admonition>
 
-## 1. Shots
+## Instantiate the Options class
+
+In the example below, we create an instance of the `Options` class. `optimization_level` is a first level option and can be passed as an input parameter. Options related to the execution environment are passed using the `environment` parameter.
+
+```python
+from qiskit_ibm_runtime import Options
+
+options = Options(optimization_level=3, environment={"log_level": "INFO"})
+```
+
+The `Options` class supports auto-complete. Once you create an instance of the `Options` class, you can use auto-complete to see what options are available. If you choose one of the categories, you can use auto-complete again to see what options are available under that category. 
```

Comment:
I know this is copied over, but it's kind of weird. Opened https://github.com/Qiskit/documentation/issues/181 to investigate in a follow up. No need to do anything now :)